### PR TITLE
Added project showcase section

### PR DESCRIPTION
**Pull request by G-R3**
I added a section to showcase some of my projects focusing on web dev. I only included 2 for now but will likely add more when I'm done with everything else. My focus is to make this page responsive which is why I'm doing a mobile first approach, something new to me. I've also used the clamp css function for font sizes and its pretty neat. I just discovered it so I'm probably using it wrong here.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -3,6 +3,7 @@ Table of Content
   1. Navbar Section
   2. Landing Section
   3. About Section
+  4. Project Section
   > Media Queries
 
   TODO: can i split this into different files? (ex: nav.css ?)
```
Comment (matty377) [inline portfolio/src/main/webapp/style.css:9]: You can, you just include them each separately with a `<link>` element.

You can also look into CSS libraries like less.css or SASS which will kind of "compile" your CSS for you. They also support inheritance and things like that. There is no requirement to do any of that but I wanted to make you aware of it.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -201,6 +203,69 @@ body {
...
+  width: 100%;
+  display: block;
+}
+.card-body {
+  width: 100%;
+  padding: 1em 2em;
+}
+.card-body .card-header {
+  /* font-size: 1.2rem; */
+  font-size: clamp(1rem, 1.4rem, 2rem);
```
Comment (matty377) [inline portfolio/src/main/webapp/style.css:243]: This usage looks correct to me. I hadn't seen this before. Very cool! - https://developer.mozilla.org/en-US/docs/Web/CSS/clamp()

**Review by matty377 — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (G-R3) [reply portfolio/src/main/webapp/style.css:9]: Got it. Will 100% be checking it out this week. Thank you!

**Pull request merged**